prow/cmd/deck/static/spyglass/lens.ts

```diff
@@ -189,6 +191,21 @@ class SpyglassImpl implements Spyglass {
     this.scrollTo(0, top).then();
   }
 
+ private setLink(match: string, attr: string): string {
+    if (typeof attr != 'undefined') {
+        return match;
+    }
+    return '</span><a target="_blank" href="' + match + '">' + match +'</a><span>';
+  }
+
+  private createHyperlinks(parent: Element): void {
+    var re = /((?:href|src)=")?(\b(https?|ftp|file):\/\/[-A-Z0-9+&@#\/%?=~_|!:,.;]*[-A-Z0-9+&@#\/%=~_|])/ig;
```

Comment:
this is a 😮  regex, can you link the stack overflow post here please?

Reply:
Done